src/go/rpk/pkg/config/schema.go

```diff
@@ -20,7 +20,7 @@ type Config struct {
 	Redpanda         RedpandaConfig         `yaml:"redpanda" mapstructure:"redpanda" json:"redpanda"`
 	Rpk              RpkConfig              `yaml:"rpk" mapstructure:"rpk" json:"rpk"`
 	Pandaproxy       *Pandaproxy            `yaml:"pandaproxy,omitempty" mapstructure:"pandaproxy,omitempty" json:"pandaproxy,omitempty"`
-	PandaproxyClient *PandaproxyClient      `yaml:"pandaproxy_client,omitempty" mapstructure:"pandaproxy_client,omitempty" json:"pandaproxyClient,omitempty"`
+	PandaproxyClient *KafkaClient           `yaml:"pandaproxy_client,omitempty" mapstructure:"pandaproxy_client,omitempty" json:"pandaproxyClient,omitempty"`
```

Comment:
don't you need to provide migration from `pandaproxy_client` -> `kafka_client` or smth like that

Reply:
> don't you need to provide migration from `pandaproxy_client` -> `kafka_client` or smth like that

No, because the serialisation formats `json`, `mapstructure`, and `yaml`, explicitly set the field name.